### PR TITLE
Added implementation for LINE Things dev board

### PR DESCRIPTION
**Pull request by cpulabs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,22 +6,23 @@ LINE Things compatible device example firmware and LIFF application code are inc
...
 
 To start developing for LINE Things using the example code and the sample firmware, the following steps are necessary.
 
 ### Installing the Firmware
 
 To start development using the example firmwares, you will need a compatible Bluetooth LE enabled development board.
-Currently LINE Things Starter firmwares supports the following 3 development boards
+Currently LINE Things Starter firmwares supports the following 6 development boards
 
+- LINE Things dev board
```
Comment (teknuze) [inline README.md]: I think it would be nice to link this to a blog post or to a page where users can know more about the hardware?

**Review by teknuze — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -6,22 +6,23 @@ LINE Things compatible device example firmware and LIFF application code are inc
...
 - [Espressif ESP32-DevKitC](https://www.espressif.com/en/products/hardware/esp32-devkitc/overview)
 - [M5Stack （ESP32)](http://m5stack.com/)
 - [Adafruit Feather nRF52 Bluefruit LE](https://www.adafruit.com/product/3406)
 - [BBC micro:bit](https://microbit.org/)
 - [Obniz](https://obniz.io/)
 
-Each firmware is located under、`esp32`, `m5stack`, `nrf52`, `microbit`, `obniz` respectively.
```
Comment (hktechn0) [inline README.md:24]: some "ゴミ" bytes after microbit
 `microbit�, obniz`
Could you remove it?

File: line-things-dev-board/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+* Micro-USB to USB Cable
+
+## Installation
+Please ensure you have Arduino IDE installed and the board **disconnected**.
+
+1. Open Arduino IDE
+2. Go into **Preferences**
+3. Add `https://www.adafruit.com/package_adafruit_index.json` as an 'Additional Board Manager URL'
+4. Go to **Boards Manager** from the Tools -> Board menu
+5. Search for nRF52 and install **Adafruit nRF52 by Adafruit** (version 0.8.6).
```
Comment (hktechn0) [inline line-things-dev-board/README.md]: Maybe we can update version of Adafruit library. 0.8.6 is old.

**Review by hktechn0 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cpulabs) [reply README.md]: I added that.

File: README.md (hunk 2 above)
Comment (cpulabs) [reply README.md:24]: removed ゴミ

File: line-things-dev-board/README.md (hunk 3 above)
Comment (cpulabs) [reply line-things-dev-board/README.md]: Changed to version 0.9.3

**Pull request merged**